sources/packages/api/src/database/migrations/1653426583147-CleanEducationProgramOfferingAddRemoveCol.ts

```diff
@@ -0,0 +1,19 @@
+import { getSQLFileData } from "../../utilities";
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class CleanEducationProgramOfferingAddRemoveCol1653426583147
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(getSQLFileData("", "EducationProgramsOfferings"));
```

Comment:
file name is blank here.

Reply:
The file has to be deleted.

Reply:
I didn't get this point. Can we connect?